src/main/kotlin/fi/metatavu/metaform/server/persistence/dao/MetaformDAO.kt

```diff
@@ -144,7 +145,7 @@ class MetaformDAO : AbstractDAO<Metaform>() {
    * @param visibility visibility
    * @return list of Metaforms
    */
-  fun listByVisibility(visibility: MetaformVisibility): List<Metaform> {
```

Comment:
DAO methods should reflect the exact functionality, so rename this to listByVisibilityAndDeleted